### PR TITLE
Change background color of  MIXED sequencing tag on cases page

### LINKED ISSUE #3434: Invisible "mixed" analysis type in light mode
In the list of cases, of case individuals contain different analyses types, then the tag "mixed" becomes invisible in light mode:

image

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3434

Light mode, from this:

image

To this: **THIS IS THE ACTUAL FIX**

image

Dark mode, from this:

image

To this:

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -84,1785 +85,4 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Unfreeze Werkzeug lib since Flask_login v.0.6 with bugfix has been released
 - Sort gene panels by name (panelS and variant page)
 - Removed unused `server.blueprints.alignviewers.unindexed_remote_static` endpoint
-- User sessions to check files served by `server.blueprints.alignviewers.remote_static` endpoint
```
Comment (dnil) [inline CHANGELOG.md:87]: Oopsie?

File: scout/server/blueprints/institutes/templates/overview/cases.html [hunk 2]
```diff
@@ -158,7 +158,7 @@
         {% elif analysis_type == 'panel' %}
           {% set badge_class = 'info' %}
         {% elif analysis_type == 'mixed' %}
-          {% set badge_class = 'mixed' %}
+          {% set badge_class = 'dark' %}
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/cases.html:161]: 👍

**Review by dnil — CHANGES_REQUESTED** [review]
Nooo, don't kill the changelog!

File: CHANGELOG.md (hunk 1 above)
Comment (northwestwitch) [reply CHANGELOG.md:88]: Ah, weird, I'll reintroduce it, thanks!

**Review by dnil — APPROVED** [review]
👍

**Pull request merged**